src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapListMessagesBodyStructureScenario.scala

```diff
@@ -0,0 +1,25 @@
+package org.apache.james.gatling.imap.scenari
+
+import com.linagora.gatling.imap.PreDef._
+import com.linagora.gatling.imap.protocol.command.FetchAttributes.AttributeList
+import com.linagora.gatling.imap.protocol.command.MessageRange.From
+import com.linagora.gatling.imap.protocol.command.MessageRanges
+import io.gatling.core.Predef._
+import io.gatling.core.feeder.FeederBuilder
+import io.gatling.core.structure.ScenarioBuilder
+
+import scala.concurrent.duration._
+
+class ImapListMessagesBodyStructureScenario {
+
+  val listMessagesBodyStructure = exec(imap("select").select("INBOX").check(ok))
+    .exec(imap("fetch").fetch(MessageRanges(From(1)), AttributeList("BODY")).check(ok))
```

Comment:
We should fetch explicitly 1000 messages and maybe ensure they are not contiguous by selecting arbitrary range

Reply:
The issue is that in our stress test provisioning mailboxes have 260-320 messages per mailbox.

I'm a bit stuck, do you have any strategy for this?

Reply:
I suggest you grab the `msn` range returned by the SELECT operation and create an arbitrary sample of the `msn`  (say with 100 messages for now). We should than complete the operation in 1/10th of the original timing, so let's say 1 second.